tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -158,7 +165,7 @@ def __eq__(self, other):
             return False
         return True
 
-    def __hash__(self):
+    def __hash__(self) -> int:
```

Comment:
May want to derive this class from [`abc.Hashable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Hashable)